src/main/java/net/snowflake/client/core/ResultUtil.java

```diff
@@ -212,11 +210,10 @@ public static SFTimestamp getSFTimestamp(
    *
    * @param obj time object
    * @param scale time scale
-   * @param session session object
    * @return snowflake time object
    * @throws SFException if time is invalid
    */
-  public static SFTime getSFTime(String obj, int scale, SFBaseSession session) throws SFException {
+  public static SFTime getSFTime(String obj, int scale) throws SFException {
```

Comment:
Please check with Java Stored proc team.Please find why it was added initially this way. Changing public method may cause some regressions.

Reply:
Good point. I reverted the changes to public API.